### PR TITLE
Add logo and "version" to `metadata.json`

### PR DESCRIPTION
**Pull request by CasperWA**
Version is determined based on commits.
Minor version was upped latest for the commit by @ltalirz on Oct 22, 2018.
Patch version is the sum of the following commits.

Updated header for README.md

Added LSMO logo from https://lsmo.epfl.ch/

### CODE REVIEW

**Comment by CasperWA** [comment]
Should this repo be moved to aiidalab?

**Comment by ltalirz** [comment]
I leave both merging this & the decision of where to host this to @yakutovicha

**Comment by yakutovicha** [comment]
> Should this repo be moved to aiidalab?

I am not sure if it is the right place. I would rather move it to the [LSMO](https://github.com/lsmo) repository.

I think aiidalab is more for the general purpose tools.

@ltalirz @giovannipizzi what do you think?

File: metadata.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
     "description": "Applications from the LSMO group",
-    "title": "LSMO apps"
+    "title": "LSMO apps",
+    "version": "0.8.6",
```
Comment (yakutovicha) [inline metadata.json]: Just replace version with "0.1.0", I've never actually released this package.

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks @CasperWA , just modify the version and I will approve

File: metadata.json (hunk 1 above)
Comment (CasperWA) [reply metadata.json]: Done

**Pull request merged**